--- redis.conf
+++ chillbox.redis.conf
@@ -135,7 +135,7 @@
 
 # Accept connections on the specified port, default is 6379 (IANA #815344).
 # If port 0 is specified Redis will not listen on a TCP socket.
-port 6379
+port 0
 
 # TCP listen() backlog.
 #
@@ -152,8 +152,8 @@
 # incoming connections. There is no default, so Redis will not listen
 # on a unix socket when not specified.
 #
-# unixsocket /run/redis.sock
-# unixsocketperm 700
+unixsocket /run/redis.sock
+unixsocketperm 700
 
 # Close the connection after a client is idle for N seconds (0 to disable)
 timeout 0
@@ -407,7 +407,7 @@
 # {unixsocket}      Unix domain socket listening on, or "".
 # {config-file}     Name of configuration file used.
 #
-proc-title-template "{title} {listen-addr} {server-mode}"
+proc-title-template "{title} {listen-addr} {unixsocket} {server-mode}"
 
 ################################ SNAPSHOTTING  ################################
 
@@ -501,7 +501,7 @@
 # The Append Only File will also be created inside this directory.
 #
 # Note that you must specify a directory here, not a file name.
-dir ./
+dir /var/lib/redis
 
 ################################# REPLICATION #################################
 
@@ -1022,7 +1022,7 @@
 # The format of the external ACL user file is exactly the same as the
 # format that is used inside redis.conf to describe users.
 #
-# aclfile /etc/redis/users.acl
+aclfile /etc/redis/users.acl
 
 # IMPORTANT NOTE: starting with Redis 6 "requirepass" is just a compatibility
 # layer on top of the new ACL system. The option effect will be just setting
@@ -1116,7 +1116,7 @@
 # limit for maxmemory so that there is some free RAM on the system for replica
 # output buffers (but this is not needed if the policy is 'noeviction').
 #
-# maxmemory <bytes>
+maxmemory 2mb
 
 # MAXMEMORY POLICY: how Redis will select what to remove when maxmemory
 # is reached. You can select one from the following behaviors:
@@ -1147,6 +1147,8 @@
 #
 # maxmemory-policy noeviction
 
+maxmemory-policy volatile-ttl
+
 # LRU, LFU and minimal TTL algorithms are not precise algorithms but approximated
 # algorithms (in order to save memory), so you can tune it for speed or
 # accuracy. By default Redis will check five keys and pick the one that was
